.lighthouserc.js

```diff
@@ -0,0 +1,234 @@
+               'render-blocking-resources': ['warn', {minScore: 0}], // failing
+               'unminified-css': ['warn', {minScore: 0}], // failing 
+               'unminified-javascript': ['warn', {minScore: 0}], // failing
+               'unused-css-rules': ['warn', {minScore: 0}], // failing
+               'uses-text-compression': ['warn', {minScore: 0}], // failing
+               // Best Practice Audits
+               'uses-http2': ['error', {minScore: 0}], // failing
+               'errors-in-console': ['error', {minScore: 0}], //failing
+  
+            }
+          },
+          {
+            matchingUrlPattern: 'http://[^/]+/about',
+            assertions: {
+               // Performance Opportunities
+               'render-blocking-resources': ['warn', {minScore: 0}], // failing
+               'unminified-css': ['warn', {minScore: 0}], // failing 
+               'unminified-javascript': ['warn', {minScore: 0}], // failing
+               'unused-css-rules': ['warn', {minScore: 0}], // failing
+               'uses-text-compression': ['warn', {minScore: 0}], // failing
+               // Best Practice Audits
+               'uses-http2': ['error', {minScore: 0}], // failing
+               'errors-in-console': ['error', {minScore: 0}], //failing
+  
+            }
+          },
+          {
+            matchingUrlPattern: 'http://[^/]+/terms',
+            assertions: {
+               // Performance Opportunities
+               'render-blocking-resources': ['warn', {minScore: 0}], // failing
+               'unminified-css': ['warn', {minScore: 0}], // failing 
+               'unminified-javascript': ['warn', {minScore: 0}], // failing
+               'unused-css-rules': ['warn', {minScore: 0}], // failing
+               'uses-text-compression': ['warn', {minScore: 0}], // failing
+               // Best Practice Audits
+               'uses-http2': ['error', {minScore: 0}], // failing
+               'errors-in-console': ['error', {minScore: 0}], //failing
+  
+            }
+          },
+          {
+            matchingUrlPattern: 'http://[^/]+/thanks',
+            assertions: {
+               // Performance Opportunities
+               'render-blocking-resources': ['warn', {minScore: 0}], // failing
+               'unminified-css': ['warn', {minScore: 0}], // failing 
+               'unminified-javascript': ['warn', {minScore: 0}], // failing
+               'unused-css-rules': ['warn', {minScore: 0}], // failing
+               'uses-text-compression': ['warn', {minScore: 0}], // failing
+               // Best Practice Audits
+               'uses-http2': ['error', {minScore: 0}], // failing
+               'errors-in-console': ['error', {minScore: 0}], //failing
+            }
+          },
+        ]
+      },
+      upload: {
+        target: 'temporary-public-storage'
+      },
+      server: {
+        // server options here
+      },
+      wizard: {
+        // wizard options here
+      },
```

Comment:
Could you remove these?

Reply:
Yes, I can, I left them there in case we want to add server or wizard options in the future. Should I remove them?

Reply:
Yep, maybe add some explanation what wizard has as options, but remove it anyway, we can always add them later.

Reply:
Done